packages/core/js-client/src/connection/RelayConnection.ts

```diff
@@ -180,19 +180,27 @@ export class RelayConnection implements IConnection {
       );
     }
 
-    log.trace("sending particle...");
-
     // Reusing active connection here
     const stream = await this.lib2p2Peer.dialProtocol(
       this.relayAddress,
       PROTOCOL_NAME,
     );
 
-    log.trace("created stream with id ", stream.id);
+    log.trace(
+      "sending the particle %s to the created stream %s",
+      particle.id,
+      stream.id,
+    );
+
     const sink = stream.sink;
 
     await pipe([fromString(serializeToString(particle))], encode, sink);
```

Comment:
is there never an error here? 🤔 Or is there a log for particle sending error in a separate place?

Reply:
Yes, particle log for network error resides in calling class - `FluencePeer`

Reply:
https://github.com/fluencelabs/js-client/blob/15f96dc5f78693ac9bb86f21b79ed035ffd97977/packages/core/js-client/src/jsPeer/FluencePeer.ts#L360C26-L360C26